src/desktop/extension.ts

```diff
@@ -25,7 +25,7 @@ import { registerChatParticipant } from '@/chat';
 import { registerUnsupervisedActions, registerUnsupervisedHandlers } from './unsupervised';
 import { registerSendSelectionToNeuro } from '@/editing';
 
-export function activate(context: vscode.ExtensionContext) {
+export async function activate(context: vscode.ExtensionContext) {
```

Comment:
Do not mark activate functions as async.

```suggestion
export function activate(context: vscode.ExtensionContext) {
```